pkgs/desktops/plasma-5/plasma-nm/default.nix

```diff
@@ -14,6 +14,10 @@ plasmaPackage {
       src = ./0001-mobile-broadband-provider-info-path.patch;
       inherit mobile_broadband_provider_info;
     })
+    (substituteAll {
+      src = ./0002-openvpn-binary-path.patch;
+      inherit openvpn;
+    })
```

Comment:
I'm not sure that `makeWrapper` will do the job here. `plasma-nm` provides several plugins (plasmoids) and there is no guarantee which executable will load each.

Adding it to `propagatedBuildInputs` will not work.

I would prefer the path be patched into the source code as with other executables.